recipes/mentalist/post-link.sh

```diff
@@ -0,0 +1 @@
+ln -s $PREFIX/bin/MentaLiST.jl $PREFIX/bin/mentalist
```

Comment:
Why is this done post install and not in the build script?

Reply:
No good reason. Moved it to `build.sh`.